src/udev/udev-builtin-blkid.c

```diff
@@ -321,8 +321,8 @@ static int builtin_blkid(struct udev_device *dev, int argc, char *argv[], bool t
         if (is_gpt)
                 find_gpt_root(dev, pr, test);
 
-        blkid_free_probe(pr);
 out:
+        blkid_free_probe(pr);
```

Comment:
I was wrong. The leak had been here when d47f6ca5f9b7a0b40 was merged. Sorry for the misleading comment.